docs/parca-agent-language-support.md

```diff
@@ -19,7 +19,7 @@ All JIT compiled languages with frame pointer, and [Perf map](https://github.com
 * [.NET](https://github.com/parca-dev/parca-demo/tree/main/dotnet)
 * [Deno](https://github.com/parca-dev/parca-demo/tree/main/deno)
 * [Erlang](https://github.com/parca-dev/parca-demo/tree/main/erlang)
-* [Java](https://github.com/parca-dev/parca-demo/tree/main/java)
+* [Java](docs/java-support)
```

Comment:
This is what's causing the build to fail (docusaurus correctly detected that this link would be broken as is as it's relative to what is already the `/docs` path. I think this either needs to be just `java-support` or `/docs/java-support`.

Reply:
Oops, my bad! I was overconfident on my Markdown skills so didn't test locally in the morning. :see_no_evil: